rskj-core/src/test/java/co/rsk/test/builders/P2shErpFederationBuilder.java

```diff
@@ -0,0 +1,79 @@
+package co.rsk.test.builders;
+
+import co.rsk.bitcoinj.core.BtcECKey;
+import co.rsk.bitcoinj.core.NetworkParameters;
+import co.rsk.peg.bitcoin.BitcoinTestUtils;
+import co.rsk.peg.federation.ErpFederation;
+import co.rsk.peg.federation.FederationArgs;
+import co.rsk.peg.federation.FederationFactory;
+import co.rsk.peg.federation.FederationMember;
+import co.rsk.peg.federation.constants.FederationConstants;
+
+import java.time.Instant;
+import java.util.List;
+
+public class P2shErpFederationBuilder {
+    private List<BtcECKey> membersPublicKeys;
+    private List<BtcECKey> erpPublicKeys;
+    private long erpActivationDelay;
+    private Instant creationTime;
+    private long creationBlockNumber;
+    private NetworkParameters networkParameters;
+
+    public P2shErpFederationBuilder() {
+        this.membersPublicKeys = BitcoinTestUtils.getBtcEcKeysFromSeeds(
+            new String[]{"member01", "member02", "member03", "member04", "member05", "member06", "member07", "member08", "member09"}, true
+        );
+        this.erpPublicKeys = BitcoinTestUtils.getBtcEcKeysFromSeeds(
+            new String[]{"erp01", "erp02", "erp03", "erp04"}, true
+        );
+        this.erpActivationDelay = 52_560;
+        this.creationTime = Instant.ofEpochMilli(0);
+        this.creationBlockNumber = 0;
+        this.networkParameters = NetworkParameters.fromID(NetworkParameters.ID_MAINNET);
+    }
+
+    public P2shErpFederationBuilder withFederationConstants(FederationConstants federationConstants) {
```

Comment:
Not sure about this. I'm not a fan of having 2 separate way of setting the same value, it might lead to contradictions. I'd just stick to `withErpPublicKeys` and `withErpActivationDelay`

Reply:
yeah you right, i thought it could save us effort but to prioritize consistency sounds better